server/models/UserToAt.js

```diff
@@ -1,5 +1,5 @@
 module.exports = function(sequelize, DataTypes) {
-    return sequelize.define(
+    let UserToAt = sequelize.define(
```

Comment:
Nonblocking: I understand why `UserToAt` is returned here, but for `AtName`, I don't understand.

Reply:
oh right.. I thought I was going to add associations to the ATName model but didn't, so that change snuck in. It has no effort, though.